src/Mod/Assembly/CommandInsertLink.py

```diff
@@ -157,7 +166,7 @@ def accept(self):
 
             if translation != App.Vector():
                 commands = commands + (
-                    f"item.Placement.base = App.Vector({translation.x}."
+                    f"item.Placement.base = App.Vector({translation.x},"
```

Comment:
I don't even understand how it was working with this typo...
And indeed when I look at the python printout : 
```
>>> item.LinkedObject = App.ActiveDocument.getObject("Body003")
>>> item.Label = "Body"
>>> item.Placement.base = App.Vector(4.5.4.5,4.5)
```
Yet interestingly this does not seem to have any effect see my inserted cube are correctly positioned: 
<img width="956" height="634" alt="image" src="https://github.com/user-attachments/assets/f68f9f1f-31f9-41b1-84b3-7ea7db15fc42" />

Reply:
Besides the user may move the objects manually during the insertion. So this python code would be off in this case.

So my conclusion is that these python commands are only for the printout for macro recording but they don't actually get executed.